### ISSUE TITLE
Missing firmware 

### ISSUE THREAD

**Issue by asharkinwater** [TTN-/IcePick-Delta#3]
Hey TTN, 
Great project! I really like your design and am considering making a similar style delta. I was wondering if you could upload the firmware for this project or point me in the direction of where I can find it. It seems that the [page]( https://github.com/Laura3/Marlin/tree/Marlin_v1) you provided no longer exists. 

Thank you for your time, 
August

**Comment by TTN-** [comment]
No worries mate. I'll upload it tonight :-)

**Comment by TTN-** [comment]
@asharkinwater thanks for letting me know the link got broken. I've added it to the repo here in the firmware folder. From memory it should just work, though its been a few years: https://github.com/TTN-/IcePick-Delta/tree/master/Firmware

I do have to warn you calibrating these things is a pain in the butt, I got it narrowed down to 0.2mm accuracy but only after I machined the pulleys round on a lathe (my badly built printer at the time didn't produce round circles).. You need very good precision to get it right. Good fun though :-) Let me know how you go!

**Comment by asharkinwater** [comment]
Will do! I Just tested the firmware on a ramps board and some steppers I had laying around and everything seems to be working just fine.  I look forward to building one over the next month or so.  Cheers!

**Comment by TTN-** [comment]
Files uploaded as requested, issue resolved :)
If there's anything not resolved please leave a comment or issue another ticket
